404.php

```diff
@@ -47,10 +47,10 @@
 					<?php
 					/* translators: %1$s: smiley */
 					$archive_content = '<p>' . sprintf( __( 'Try looking in the monthly archives. %1$s', 'lean' ), convert_smilies( ':)' ) ) . '</p>';
-					the_widget( 'WP_Widget_Archives', 'dropdown=1', "after_title=</h2>$archive_content" );
```

Comment:
Hey @nucro don't change nothing of PHP, basic styles means only sass files, please keep things simple and focus only on 1 task at the time.

Thank you.